### ISSUE TITLE
[Bug] Welcome page cannot be next

### ISSUE THREAD

**Issue by Jesse205** [Mahmud0808/ColorBlendr#35]
```
FATAL EXCEPTION: main
Process: com.drdisagree.colorblendr, PID: 7103
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.drdisagree.colorblendr.ui.widgets.SelectableViewWidget.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
	at sh0.Q0(Unknown Source:141)
	at androidx.fragment.app.Fragment.t1(Unknown Source:19)
	at androidx.fragment.app.p.e(Unknown Source:128)
	at androidx.fragment.app.p.j(Unknown Source:135)
	at androidx.fragment.app.m.z(Unknown Source:574)
	at androidx.fragment.app.m.S(Unknown Source:81)
	at androidx.fragment.app.m.y(Unknown Source:31)
	at androidx.viewpager2.adapter.a.L(Unknown Source:193)
	at androidx.viewpager2.adapter.a.A(Unknown Source:2)
	at ae.a(Unknown Source:40)
	at androidx.recyclerview.widget.e.l(Unknown Source:217)
	at androidx.recyclerview.widget.LinearLayoutManager.i1(Unknown Source:33)
	at androidx.recyclerview.widget.LinearLayoutManager.V0(Unknown Source:48)
	at androidx.recyclerview.widget.LinearLayoutManager.n1(Unknown Source:34)
	at androidx.recyclerview.widget.LinearLayoutManager.A0(Unknown Source:7)
	at androidx.recyclerview.widget.RecyclerView.f0(Unknown Source:23)
	at androidx.recyclerview.widget.RecyclerView.e0(Unknown Source:23)
	at androidx.recyclerview.widget.RecyclerView.onTouchEvent(Unknown Source:377)
	at l81.onTouchEvent(Unknown Source:6)
	at android.view.View.dispatchTouchEvent(View.java:14955)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3329)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2990)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3343)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:3004)
	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:898)
	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2077)
	at android.app.Activity.dispatchTouchEvent(Activity.java:4431)
	at h5.dispatchTouchEvent(Unknown Source:2)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:817)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:805)
	at android.view.View.dispatchPointerEvent(View.java:15242)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:8312)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:8042)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7436)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:7493)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:7459)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:7641)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:7467)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:7698)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7440)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:7493)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:7459)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:7467)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7440)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:10924)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:10826)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:10779)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:11177)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:310)
	at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
	at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:283)
	at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:11083)
	at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:11369)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1592)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1256)
	at android.view.Choreographer.doFrame(Choreographer.java:1117)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1542)
	at android.os.Handler.handleCallback(Handler.java:966)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:293)
	at android.app.ActivityThread.loopProcess(ActivityThread.java:9961)
	at android.app.ActivityThread.main(ActivityThread.java:9950)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:586)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1240)
```

**Comment by Mahmud0808** [comment]
Can you provide screen recording please?

**Comment by Jesse205** [comment]
https://github.com/Mahmud0808/ColorBlendr/assets/51242302/566efacd-d23c-4e25-ac16-6cc0767797a3

**Comment by Mahmud0808** [comment]
Is it also happening with older builds?

**Comment by Jesse205** [comment]
Yes, 1.0b8 does not have this bug. But I can't continue because my device is not root

**Comment by Mahmud0808** [comment]
[ColorBlendr v1.1.zip](https://github.com/Mahmud0808/ColorBlendr/files/14218283/ColorBlendr.v1.1.zip)

Let me know if this works or not.

**Comment by Jesse205** [comment]
> [ColorBlendr v1.1.zip](https://github.com/Mahmud0808/ColorBlendr/files/14218283/ColorBlendr.v1.1.zip)
> 
> Let me know if this works or not.

It's crashing when it starts

```
FATAL EXCEPTION: main
Process: com.drdisagree.colorblendr, PID: 18701
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.drdisagree.colorblendr.ui.widgets.SelectableViewWidget.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
	at sh0.Q0(Unknown Source:141)
	at androidx.fragment.app.Fragment.t1(Unknown Source:19)
	at androidx.fragment.app.p.e(Unknown Source:128)
	at androidx.fragment.app.p.j(Unknown Source:135)
	at androidx.fragment.app.m.z(Unknown Source:574)
	at androidx.fragment.app.m.S(Unknown Source:81)
	at androidx.fragment.app.m.y(Unknown Source:31)
	at androidx.viewpager2.adapter.a.L(Unknown Source:193)
	at androidx.viewpager2.adapter.a.A(Unknown Source:2)
	at ae.a(Unknown Source:40)
	at androidx.recyclerview.widget.e.l(Unknown Source:217)
	at androidx.recyclerview.widget.LinearLayoutManager.i1(Unknown Source:33)
	at androidx.recyclerview.widget.LinearLayoutManager.V0(Unknown Source:48)
	at androidx.recyclerview.widget.LinearLayoutManager.n0(Unknown Source:818)
	at androidx.recyclerview.widget.RecyclerView.u(Unknown Source:60)
	at androidx.recyclerview.widget.RecyclerView.s(Unknown Source:71)
	at androidx.recyclerview.widget.RecyclerView.onLayout(Unknown Source:5)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at androidx.viewpager2.widget.ViewPager2.onLayout(Unknown Source:60)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at androidx.constraintlayout.widget.ConstraintLayout.onLayout(Unknown Source:61)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:359)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:295)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:359)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:295)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1870)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1714)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1623)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:359)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:295)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1870)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1714)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1623)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:359)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:295)
	at com.android.internal.policy.DecorView.onLayout(DecorView.java:1319)
	at android.view.View.layout(View.java:24332)
	at android.view.ViewGroup.layout(ViewGroup.java:6758)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:5200)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:4528)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:3132)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:11095)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1592)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1256)
	at android.view.Choreographer.doFrame(Choreographer.java:1125)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1542)
	at android.os.Handler.handleCallback(Handler.java:966)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:293)
	at android.app.ActivityThread.loopProcess(ActivityThread.java:9961)
	at android.app.ActivityThread.main(ActivityThread.java:9950)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:586)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1240)
```

**Comment by Mahmud0808** [comment]
It was crashing because of a mistake I did in landscape layout. It is fixed in fb28758fa3cda1eb782fbc6970d215546fb48662

Kindly use [this version](https://github.com/Mahmud0808/ColorBlendr/files/14220130/ColorBlendr.v1.1.zip) for now.
